lib/private/Config/UserConfig.php

```diff
@@ -75,6 +77,7 @@ public function __construct(
 		protected IConfig $config,
 		protected LoggerInterface $logger,
 		protected ICrypto $crypto,
+		protected IEventDispatcher $dispatcher
```

Comment:
```suggestion
		protected IEventDispatcher $dispatcher,
```
I think `composer run cs:fix` would fix it as well.